### PR TITLE
Translation: measuring_success

### PR DESCRIPTION
**Pull request by wgzhao**

### CODE REVIEW

**Comment by wgzhao** [comment]
The response  is so slow that I had to close this PR

**Comment by Cubik65536** [comment]
> The response is so slow that I had to close this PR

@wgzhao Everyone in this team are busy, no one can promise that we can finish review in a short time, we are still managing to get more time for content review, just be patient please.

File: translation/Measuring Success/measuring_success.md [hunk 1]
```diff
@@ -1,46 +1,25 @@
...
-
-This section is useful for you if:
-
-- You want to know if your community is healthy.
-- You are seeking data that can demonstrate successes and failures.
-- You need more contributors in your project.
-- You have read this far, why stop now?
+## 如何衡量成功
+
+在开源之前，衡量一个软件工作的成功与否往往是以严格的财务术语来定义的。一个公司卖出了多少盒软件--是的，多少盒？这笔收入被支持、编写和分发的成本所抵消（同样是盒子），仅此而已。利润就是衡量成功的标准。
```
Comment (KevinZonda) [inline translation/Measuring Success/measuring_success.md:3]:
```suggestion
在开源之前，衡量一个软件工作的成功与否往往是以严格的财务术语来定义的。一个公司卖出了多少盒软件──是的，多少盒？这笔收入被支持、编写和分发的成本所抵消（同样是盒子），仅此而已。利润就是衡量成功的标准。
```

File: translation/Measuring Success/measuring_success.md [hunk 2]
```diff
@@ -1,46 +1,25 @@
...
+
+在开源的世界里，情况并不完全如此。首先，虽然存在一些商业成功的开源项目，但这些项目的成功仍然不是严格地与财务状况挂钩的。成功也不是以下载量这个共同的基准来设定的。例如，我们很容易假定，如果一个软件项目有数百万的下载量，那么它就做得非常成功。但是这仅仅只是用户在稳定地消费该软件，并不意味着该项目社区中没有内部问题。依旧可能存在任何因素会使项目停顿下来。相反，有许多非常成功的项目并没有头条新闻和巨大的人气。
```
Comment (KevinZonda) [inline translation/Measuring Success/measuring_success.md:5]: `依旧可能存在任何因素会使项目停顿下来` is weird.

File: translation/Measuring Success/measuring_success.md [hunk 3]
```diff
@@ -1,46 +1,25 @@
...
+
+那么，如果金钱和名声不是开源软件项目的成功因素，什么才是呢？
+
+对于开源项目来说，成功不仅仅是用商业或消费方面的指标来衡量的。由于分布式版本控制和协作工具的普及，自由和开源项目的发展变得更加透明。虽然这些工具，如基于 git 的工具，肯定给贡献者带来了把项目放在一起的巨大优势，但另一个好处是，它们也允许对项目的表现进行更有力的定量观察。
```
Comment (KevinZonda) [inline translation/Measuring Success/measuring_success.md:9]: `虽然这些工具，如基于 git 的工具，肯定给贡献者带来了把项目放在一起的巨大优势，但另一个好处是，它们也允许对项目的表现进行更有力的定量观察。` is still weird. Coul try think a better word to instead `虽然`.

**Review by KevinZonda — CHANGES_REQUESTED** [review]
I have no idea. /cc @Cubik65536

**Comment by KevinZonda** [comment]
Apologise to slow CR. Currently I am busier than usual and under extremely high pressure, it may take longer to response you. Apologise again for the inconvenience I caused. :)

File: translation/Measuring Success/measuring_success.md (hunk 1 above)
Comment (Cubik65536) [reply translation/Measuring Success/measuring_success.md:3]: 而且 `多少盒软件` 其实挺奇怪的，是否可以使用：

```suggestion
在开源之前，衡量一个软件工作的成功与否往往是以严格的财务术语来定义的。一个公司卖出了多少份软件──是的，多少份？这笔收入被支持、编写和分发（同样是多少份）的成本所抵消，仅此而已。利润就是衡量成功的标准。
```

File: translation/Measuring Success/measuring_success.md (hunk 2 above)
Comment (Cubik65536) [reply translation/Measuring Success/measuring_success.md:5]:
```suggestion
在开源的世界里，情况并不完全如此。首先，虽然存在一些商业成功的开源项目，但这些项目的成功仍然不是严格地与财务状况挂钩的。成功也不是以下载量这个共同的基准来设定的。例如，我们很容易假定，如果一个软件项目有数百万的下载量，那么它就做得非常成功。但是这仅仅只是用户在稳定地消费该软件，并不意味着该项目社区中没有内部问题。可能会使项目停摆的问题仍然有可能存在。相反，有许多非常成功的项目并没有头条新闻和巨大的人气。
```

File: translation/Measuring Success/measuring_success.md [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
+在过去，我们很难实时看到一个特定项目的过程是否成功。你会看到一个项目努力的产出，但不知道这个过程在达到目的的过程中采取的步骤。至少，在没有大量努力的情况下是如此。因此，建立一个项目的健康和成功更多的是用主观的方法，通常是基于已经在许多文化中根深蒂固的社会学和商业导向原则。不过现在，你可以很容易地潜入开放开发所展示的数据，并使用这些数据来得出关于项目健康的定量信息。本章所探讨的正是这些概念。
 
-This section contains chapters that:
+本章包含以下部分：
 
-- Discuss the formation of healthy, participatory communities.
-- Examine how metrics can be applied to projects.
-- Determine which metrics will be the most useful for a project.
-- Highlight ways to increase participation in a project’s announcements.
+- 讨论健康的、参与性社区的形成
```
Comment (Cubik65536) [inline translation/Measuring Success/measuring_success.md:15]:
```suggestion
- 讨论健康的、参与度高的社区的形成
```

File: translation/Measuring Success/measuring_success.md (hunk 3 above)
Comment (Cubik65536) [reply translation/Measuring Success/measuring_success.md:9]: Still have to reconsider the translation for this part

**Review by Cubik65536 — CHANGES_REQUESTED** [review]

**Comment by wgzhao** [comment]
> Apologise to slow CR. Currently I am busier than usual and under extremely high pressure, it may take longer to response you. Apologise again for the inconvenience I caused. :)

I'm sorry for that,  I take back what I said before.

File: translation/Measuring Success/measuring_success.md (hunk 3 above)
Comment (wgzhao) [reply translation/Measuring Success/measuring_success.md:9]: sure

File: translation/Measuring Success/measuring_success.md (hunk 1 above)
Comment (KevinZonda) [reply translation/Measuring Success/measuring_success.md:3]: Looks better

File: translation/Measuring Success/measuring_success.md (hunk 2 above)
Comment (KevinZonda) [reply translation/Measuring Success/measuring_success.md:5]: LGTM

**Comment by KevinZonda** [comment]
> > Apologise to slow CR. Currently I am busier than usual and under extremely high pressure, it may take longer to response you. Apologise again for the inconvenience I caused. :)
> 
> I'm sorry for that, I take back what I said before.

Thanks for your understanding. :)

File: translation/Measuring Success/measuring_success.md (hunk 4 above)
Comment (wgzhao) [reply translation/Measuring Success/measuring_success.md:15]: 改成 高参与性，读起来顺口一些

**Comment by KevinZonda** [comment]
image

Ohhh lao tian ye a, ni kan, fa sheng le shen me?

**Comment by KevinZonda** [comment]
@wgzhao Actually, you should **NOT** use force push, this will cause all of our reviews out of date and do not work.

**Comment by wgzhao** [comment]
> 
> Ohhh lao tian ye a, ni kan, fa sheng le shen me?

As a PR commit habit, I squashed multiple commits into one

Do you see the last commit ?

**Comment by KevinZonda** [comment]
> > Ohhh lao tian ye a, ni kan, fa sheng le shen me?
> 
> As a PR commit habit, I squashed multiple commits into one
> 
> Do you see the last commit ?

Yes, it looks neat to commit, but that can only be bad for us in tracking down problems. And in the meantime, this can be done entirely at merge time, not now.

**Comment by KevinZonda** [comment]
I'll review it later, it has been 2:33 am in the UK.

+, u should add co author too.

File: translation/Measuring Success/measuring_success.md [hunk 5]
```diff
@@ -1,46 +1,25 @@
...
-
-This section is useful for you if:
-
-- You want to know if your community is healthy.
-- You are seeking data that can demonstrate successes and failures.
-- You need more contributors in your project.
-- You have read this far, why stop now?
+## 如何衡量成功
+
+在开源之前，衡量一个软件工作的成功与否往往是以严格的财务术语来定义的。一个公司卖出了多少份软件──是的，多少份？这笔收入被支持、编写和分发的成本所抵消（同样是多少份），然后就可以了。利润是衡量成功的标准。
```
Comment (Cubik65536) [inline translation/Measuring Success/measuring_success.md:3]:
```suggestion
在开源之前，衡量一个软件工作的成功与否往往是以严格的财务术语来定义的。一个公司卖出了多少份软件──是的，多少份？这笔收入被支持、编写和分发（同样是多少份）的成本所抵消，然后就可以了。利润是衡量成功的标准。
```

我觉得原文的 `（同样是多少份）` 代指分发了多少份软件，可以参考

**Review by Cubik65536 — CHANGES_REQUESTED** [review]
在 CR 过后 Force Push 是个不好的习惯，如果在 CR 之前出现了错误的 commit，FP或许是个方法，但是请不要 CR 过后 FP。FP也**不是**一个让 Pull Request 变得整洁的方法。

另外：如果你使用了其他人的 suggestion，而且手动更改并 commit 了，请记得添加 co-author，这是对别人工作的尊重

**Comment by wgzhao** [comment]
> > > Ohhh lao tian ye a, ni kan, fa sheng le shen me?
> > 
> > 
> > As a PR commit habit, I squashed multiple commits into one
> > Do you see the last commit ?
> 
> Yes, it looks neat to commit, but that can only be bad for us in tracking down problems. And in the meantime, this can be done entirely at merge time, not now.

thanks, I got it

**Comment by wgzhao** [comment]
> I'll review it later, it has been 2:33 am in the UK.
> 
> +, u should add co author too.

OK

**Review by Cubik65536 — APPROVED** [review]
Looks good to me, please look at this @KevinZonda @gh-Devin @LLLgoyour

**Comment by KevinZonda** [comment]
Sorry, currently I hv no time to CR such thing. /cc @Cubik65536 @LLLgoyour

**Comment by Cubik65536** [comment]
感谢你的贡献，我们将会合并你的更改，以后发现瑕疵再做修复。

------

鉴于你已经将你的仓库删除，我将会把你的内容手动添加到本仓库中并添加你为author

**Pull request closed without merging**